app/lib/widgets/initial/MuninLoginPage.dart

```diff
@@ -358,11 +362,16 @@ class _MuninLoginPageState extends State<MuninLoginPage> {
         Application.environmentValue.bangumiOauthClientIdentifier;
     final authorizationUrl =
         'https://${Application.environmentValue.bangumiHostForDio}'
-        '/oauth/authorize?response_type=code&client_id=$clientId';
+        '/oauth/authorize';
 
+    Map<String, String> authParam = {
+      'response_type': 'code',
+      'client_id': '$clientId',
+      'redirect_uri': Application.environmentValue.bangumiRedirectUrl,
```

Comment:
Wondering if redirect_uri is needed as it's not included before...

no action is required, I will keep an eye on this after your code is checked in.

Reply:
Yes it's really wired that it works fine with your API key (reverse egineering from release APK) but doesn't work with my API key without `redirect_uri`.